formats/cbor/commonTest/src/kotlinx/serialization/cbor/CborDefLenTest.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2017-2023 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.serialization.cbor
+
+import kotlinx.serialization.*
+import kotlin.test.*
+
+
+class CborDefLenTest {
```

Comment:
minor: would be nice to have full name here instead of shortcuts :)

Reply:
fixed in 15bbc3fcf453352b0e4d5b4a1996d64da53ff304